src/main/java/reposense/util/FileUtil.java

```diff
@@ -229,6 +237,8 @@ public static void copyDirectoryContents(String sourcePath, String outputPath, L
     }
     /**
      * Creates the {@code dest} directory if it does not exist.
+     *
+     * @throws IOException for any I/O errors in the underlying function.
```

Comment:
I think a better description would be something like `IOException if the directory could not be created.` I believe there is no need to describe where the errors come from.

Reply:
Fixed.